### PR TITLE
machinery: Ensure OneClickInstallCLI is installed

### PR DESCRIPTION
**Pull request by okurz**
Fixes problems as observed in
https://openqa.opensuse.org/tests/232806#step/machinery/3

Verified locally.

### CODE REVIEW

**Comment by coolo** [comment]
your changes to main.pm look a bit egocentric

**Comment by okurz** [comment]
fixed

File: tests/console/machinery.pm [hunk 1]
```diff
@@ -13,8 +13,9 @@ use testapi;
 
 sub run() {
     select_console 'root-console';
+    assert_script_run 'which OneClickInstallCLI || zypper -n in yast2-metapackage-handler',  200;
     assert_script_run 'yes | OneClickInstallCLI http://machinery-project.org/machinery.ymp', 200;
-    validate_script_output "machinery --help", sub { m/machinery - A systems management toolkit for Linux/ }, 100;
+    validate_script_output 'machinery --help',                                               sub { m/machinery - A systems management toolkit for Linux/ }, 100;
 }
```
Comment (coolo) [inline tests/console/machinery.pm]: wouldn't a line break be better here? :)

Comment (okurz) [inline tests/console/machinery.pm]: yes, it would but tidy will join the lines again.

**Pull request merged**